source/features/mark-merge-commits-in-list.tsx

```diff
@@ -35,23 +37,18 @@ const filterMergeCommits = async (commits: string[]): Promise<string[]> => {
 	return mergeCommits;
 };
 
-function getCommitLink(commit: HTMLElement): HTMLAnchorElement | undefined {
-	return $([
-		'a.markdown-title', // Old view style (before November 2023)
-		'.markdown-title a',
-	], commit);
-}
-
 export function getCommitHash(commit: HTMLElement): string {
-	return getCommitLink(commit)!.pathname.split('/').pop()!;
+	const hash = $(commitHashLinkInLists, commit).pathname.split('/').pop()!;
+	assertCommitHash(hash);
```

Comment:
This assertion will avoid silent issues like https://github.com/refined-github/refined-github/pull/7685#discussion_r1716273335